docs/content/integrations/dbt/reference.mdx

```diff
@@ -102,28 +102,39 @@ defs = Definitions(
 
 ## Scheduling dbt jobs
 
-Once you have your dbt assets, you can define a job that runs some or all of these assets on a schedule:
+Once you have your dbt assets, you can define a job to materialize a selection of these assets on a schedule.
 
-```python startafter=start_schedule_assets endbefore=end_schedule_assets file=/integrations/dbt/dbt.py dedent=4
-from dagster import ScheduleDefinition, define_asset_job, Definitions
+### Scheduling jobs that contain only dbt assets
 
-run_everything_job = define_asset_job("run_everything", selection="*")
+In this example, we use the <PyObject module="dagster_dbt" object="build_schedule_from_dbt_selection" /> function to create a job, `"daily_dbt_models"`, as well as a schedule which will execute this job once a day. We define the set of models we'd like to execute using [dbt's selection syntax](https://docs.getdbt.com/reference/node-selection/syntax#how-does-selection-work), in this case selecting only the models with the tag "daily".
 
-# only `order_stats` and its children
-run_something_job = define_asset_job("run_something", selection="order_stats*")
+```python startafter=start_schedule_assets_dbt_only endbefore=end_schedule_assets_dbt_only file=/integrations/dbt/dbt.py dedent=4
+from dagster_dbt import build_schedule_from_dbt_selection
 
-defs = Definitions(
-    assets=dbt_assets,
-    schedules=[
-        ScheduleDefinition(
-            job=run_something_job,
-            cron_schedule="@daily",
-        ),
-        ScheduleDefinition(
-            job=run_everything_job,
-            cron_schedule="@weekly",
-        ),
-    ],
+daily_dbt_assets_schedule = build_schedule_from_dbt_selection(
+    [my_dbt_assets],
+    job_name="daily_dbt_models",
+    cron_schedule="@daily",
+    dbt_select="tag:daily",
+)
+```
+
+### Scheduling jobs that contain both dbt assets and non-dbt assets
```

Comment:
```suggestion
### Scheduling jobs that contain dbt assets and non-dbt assets
```

Suggesting we remove `both` from this heading to cut down on length a little bit.